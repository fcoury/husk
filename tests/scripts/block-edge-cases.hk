let unit_result = {
    let x = 5;
    let y = 10;
    x + y;
};
println!("Unit result: {}", unit_result);

let nested_result = {
    let inner = {
        let a = 3;
        let b = 4;
        a * b
    };
    inner + 5
};
println!("Nested result: {}", nested_result);
