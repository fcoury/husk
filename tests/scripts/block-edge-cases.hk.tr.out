Unit result:  undefined
Nested result:  17
